Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -174,54 +174,62 @@ internal void HandleAnimTriggerMsg(int hash)
             animator.SetTrigger(hash);
         }
 
-        bool WriteParameters(NetworkWriter writer, bool autoSend)
+        uint GetDirtyBits()
```

Comment:
`GetDirtyBits`  implies this is a getter function without side effects.

But this does have side effects,   every time you call it,  it clears the dirty bits too.

perhaps we can rename it to `ResetDirtyBits`?

Reply:
How about NextParameterDeltaBits

Reply:
would be better yes.

how about:
`NextDirtyBits`

I like the word "Next"  because it implies that this moves forward

Reply:
sounds good to me!